File: Python/card.py
from payment import Payment

class Card(Payment):
    cardNUmber = int
    cvv        = int
    date       = str

    def __init__(self, paymentID, cardNUmber, cvv, date):
        super.__init__(paymentID)
        self.cardNUmber = cardNUmber
        self.cvv        = cvv
        self.date       = date

File: Java/Card.java
class Card extends Payment{
    Integer cardNumber;
    Integer cvv;
    String date;

    public Card(Integer paymentID, Integer cardNumber, Integer cvv, String date){
        super(paymentID);
        this.cardNumber = cardNumber;
        this.cvv = cvv;
        this.date = date;

    }
}